rclcpp/include/rclcpp/strategies/message_pool_memory_strategy.hpp

```diff
@@ -17,6 +17,8 @@
 
 #include <memory>
 
+#include "rosidl_runtime_cpp/traits.hpp"
```

Comment:
Seems unrelated; is this addition intentional?

Reply:
Ah, yeah I just noticed that this header would not compile by itself without this include, so basically just a missing include. It's not strictly related, but a worthwhile change, imo.